### PR TITLE
Feature/create edit

### PR DESCRIPTION
**Pull request by mischb**
updated component names and items action and components to render list name, items, store information

### CODE REVIEW

File: src/reducers/items.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      };
+
+    case ADD_ITEMS_REQUEST:
+      return { ...state, loading: true };
+
+    case ADD_ITEMS_ERROR:
+      return { ...state, error: action.error };
+
+    case ADD_ITEMS_SUCCESS:
+      for (let i = 0; i < state.items.length; i++) {
```
Comment (aseehra) [inline src/reducers/items.js]: This block could be rewritten the following ways:

```javascript
newItems = state.items.concat([action.item]);
```
or
```javascript
newItems = [...state.items, action.item];
```

File: src/store.js [hunk 2]
```diff
@@ -5,14 +5,14 @@ import { loadAuthToken } from './local-storage';
...
-import listReducer from './reducers/list';
+import listsReducer from './reducers/lists';
+import itemsReducer from './reducers/items';
 const store = createStore(
   combineReducers({
     form: formReducer,
     auth: authReducer,
-    dashboard: dashboardReducer,
-    list: listReducer,
+    lists: listsReducer,
```
Comment (aseehra) [inline src/store.js]: 💯

File: src/reducers/items.js (hunk 1 above)
Comment (aseehra) [reply src/reducers/items.js]: Oh. This might be dead/unused code at the moment. So feel free to ignore this.

File: src/components/Items.js [hunk 3]
```diff
@@ -33,11 +34,15 @@ export class List extends Component {
 
     return (
       <Fragment>
+        <NavBar />
+        <h2>{this.props.items.name}</h2>
+        <h3>
```
Comment (aseehra) [inline src/components/Items.js]: this should be wrapped in a `if (this.props.store)` block

**Comment by aseehra** [comment]
This could be closed (and not merged) since #12 builds on it and fixes some of the above errors.

**Comment by SamDaltonJr** [comment]
closing and not merging

**Pull request closed without merging**